lighthouse-core/lib/manifest-parser.js

```diff
@@ -331,10 +379,16 @@ function parsePreferRelatedApplications(jsonInput) {
   };
 }
 
+/**
+ * @param {*} jsonInput
+ */
 function parseThemeColor(jsonInput) {
   return parseColor(jsonInput.theme_color);
 }
 
+/**
+ * @param {*} jsonInput
```

Comment:
> should we be adding todos around any/* to type them later

I think for most cases, yes. There's usually no reason not to type stuff, so I think it would be good to go back and hit all our marks.

In this particular case, though, I'm not sure what the best thing is to do. We know what type we *expect* on each property, but this is taking a json file to parse it and flag anything we *don't* expect :) Technically the type should be anything serializable by JSON, so we could narrow that, at least...